test_runner.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from click.testing import CliRunner
+from runner import run, Command
+
+
+def test_should_fail_to_run_cmd_multiple_time():
+    command = Command(cmd='ping', attempts=3)
+    command.execute(repeat_times=3)
+    assert command.attempts == 0
```

Comment:
it should actually be 1 attempt :) since it failed, it counts as one.
The test will pass right now, but the code can be more correct.

If it was TDD, you would have written 1.